lib/api/controllers/documentController.js

```diff
@@ -173,6 +176,40 @@ class DocumentController extends NativeController {
     };
   }
 
+  async export (request) {
+    const { index, collection } = request.getIndexAndCollection();
+    const batchSize = request.getInteger('batchSize', 2000);
+    const query = request.getBody({});
+    const format = request.getString('format', 'jsonl');
+    const fields = request.getArray('fields', []);
```

Comment:
This should be a body array to work with http (it's the same subject as for targets list in HTTP)
For this action it's more important to have something working with HTTP since it's the targeted protocol